src/worldstate/msg/stateMsg.msg

```diff
@@ -5,3 +5,7 @@ int8 passingGate         = 2
 int8 locatingPole        = 3
```

Comment:
should be `StateMsg`, following class naming

Reply:
Changed!